### ISSUE TITLE
Feature Request: Have option to move cursor outside of paired markdown syntax

### ISSUE THREAD

**Issue by spoon-m4a** [chrisgrieser/obsidian-smarter-md-hotkeys#34]
### Feature Requested

As it is, it's a little annoying to always use `cmd + ->` to type outside of the auto paired md syntax when I use a hotkey. It'd be nice to have an option to automagically place the cursor on the other side of the text when used (either when selecting multiple words or just one). 

Bonus points if it also auto inserts a space character (though that isn't strictly necessary!)

### Which editor are you using?

- [X] New Editor
- [ ] Legacy Editor

### Checklist

- [X] I have checked that there isn't [another hotkey plugin](https://publish.obsidian.md/hub/02+-+Community+Expansions/02.01+Plugins+by+Category/Plugins+that+add+or+manage+hotkeys) that does what I want.
- [X] The feature would be useful to more users than just me.

**Comment by Gewerd-Strauss** [comment]
I agree that that would be very useful indeed. It could even be elaborated upon to do the following

```
if (cursor is in word/at word boundary)
    execute, put cursor behind the closing member of markdown syntax
else ## cursor was surrounded by whitespace
    create both members surrounding the cursor ## the same way it performs so far.
```
